base_app.htm

```diff
@@ -0,0 +1,60 @@
+<%inherit file="homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+  <%block name="navitems">
```

Comment:
I think it would be smarter to move parts of your nav-bar to the Base so that you just have specialized functions in the individual base_app

Reply:
Good idea, I've been thinking about doing that but am unsure if it's not dynamic enough